src/cli/tc.rs

```diff
@@ -37,9 +41,55 @@ pub async fn run() -> Result<()> {
         Err(e) => Err(e),
       }
     }
+    Command::Init { file_path } => Ok(init(&file_path).await?),
   }
 }
 
+pub async fn init(file_path: &str) -> Result<()> {
+  let tailcallrc = resource_str!("assets/.tailcallrc.graphql");
+
+  let ans = Confirm::new("Do you want to add a file to the project?")
+    .with_default(false)
+    .prompt();
+
+  match ans {
+    Ok(true) => {
+      println!("{}", "Enter the file name:".yellow());
+      let mut file_name = String::new();
+      io::stdin().read_line(&mut file_name)?;
+      file_name = format!("{}.graphql", file_name.trim());
+
+      let confirm = Confirm::new(&format!("Do you want to create the file {}?", file_name))
+        .with_default(false)
+        .prompt();
+
+      match confirm {
+        Ok(true) => {
+          fs::write(format!("{}/{}", file_path, &file_name), "")?;
+
+          let graphqlrc = format!(
+            r#"schema:
+- "./{}"
+- "./.tailcallrc.graphql"#,
+            file_name
+          );
+          fs::write(format!("{}/.graphqlrc.yml", file_path), graphqlrc)?;
+        }
+        Ok(false) => (),
+        Err(_) => println!("{}", "Something went wrong, please try again".red()),
+      }
+    }
+    Ok(false) => (),
+    Err(_) => println!("{}", "Something went wrong, please try again".red()),
+  }
```

Comment:
I think failures will be handled automatically because we are returning `Result`, we shouldn't use `println`